en/developer_guide/routing.rst

```diff
@@ -0,0 +1,79 @@
+Routing
+=======
+
+Base URL
+--------
+
+The page base url is compose with domain name and language prefix.
+To manage all possibilities, Open Orchestra associate each website in database with aliases that contain:
+
+* **domain**: domain name
+* **language**: language prefix
```

Comment:
There are two notion, the language which is the language in which all page are accessed and the prefix which is used to build the base url